sympy/matrices/determinant.py

```diff
@@ -476,7 +476,7 @@ def _cofactor_matrix(M, method="berkowitz"):
     minor_submatrix
     """
 
-    if not M.is_square or M.rows < 1:
+    if not M.is_square:
```

Comment:
This is fixing a bug. It is tested by the `test_inverse` tests:
```python
In [1]: M = Matrix(0, 0, [])

In [2]: M.adjugate()
---------------------------------------------------------------------------
NonSquareMatrixError 
```
The adjugate of a 0x0 matrix is again a 0x0 matrix. With the PR:
```python
In [1]: M = Matrix(0, 0, [])

In [2]: M
Out[2]: []

In [3]: M.adjugate()
Out[3]: []

In [4]: M.det()
Out[4]: 1

In [5]: M*M.adjugate() == M.det() * eye(0)
Out[5]: True
```